node/.env

```diff
@@ -0,0 +1,9 @@
+# An Example Of Environment Variables.
+
+# Airwallex API keys.
+# Use your test keys for development and live keys for real charges in production.
+
+AIRWALLEX_API_KEY=1105eacc146c413c43648e0f15764eb873810185ccea0d18e1a8b14ab97bb4e8d6cf219672c775e25f89d5267b4d1793
+AIRWALLEX_CLIENT_ID=MSKOSA7KTo2JdEnGoQmcCQ
```

Comment:
need to add this file to .gitignore as well, we don't want to show the API_KEY and CLIENT_ID even it's staging.

Reply:
If we put this in .gitignore, how can users to try a test to our node demo?